### DISCUSSION TITLE
The introduction of Google's remote inter font resulted in different rendering fonts for Mac and Windows

### DISCUSSION THREAD

**Discussion by little077** [vercel/next.js#70953] [Help]
### Summary

I don't know what the correct behavior is because I can't be sure if it's a problem on the Mac or on Windows right now
![ead1d3eb3707fb17a7f798cb1c0d2f7](https://github.com/user-attachments/assets/6f9c98c0-9c93-409a-a8a7-73a2600d9490)
![Uploading 6bf9d290e1a05625b10a5a4420d0bbc.png…]()

### Additional information

```js
I have placed it in the codebook below. Please check the final rendered font on the console and you will find that it is different
```

### Example

https://codesandbox.io/p/devbox/g75t52

**Comment by little077** [comment]
@icyJoseph I encountered a coding problem, but I don't know where the problem lies.Can you help me answer this.

**Reply by icyJoseph** [reply]
Could you try to reupload the images? One of them is broken.

So the problem you have is that the text is rendering differently, or a different font family altogether?
